src/clusterfuzz/_internal/base/tasks.py

```diff
@@ -412,6 +457,36 @@ def lease(self, _event=None):  # pylint: disable=arguments-differ
     track_task_end()
 
 
+class UTaskMainPubSubTask(PubSubTask):
+
+  def __init__(self, pubsub_message):
+    self._pubsub_message = pubsub_message
+    grandparent_class = super(PubSubTask, self)
+    grandparent_class.__init__(
+        self.attribute('command'),
+        self.attribute('argument'),
+        self.attribute('job'),
+        input_url=self.attribute('input_url'))
+
+
+def initialize_task(message):
+  """Creates a task from |messages|."""
+  if message.attributes.get('eventType', None) == 'OBJECT_FINALIZE':
```

Comment:
If `message.attributes` is a dict, then you can skip the second argument to `get()`:

```suggestion
  if message.attributes.get('eventType') == 'OBJECT_FINALIZE':
```